src/Observable.ts

```diff
@@ -127,7 +20,7 @@ export type ObservableInput<T> = ObservableOrPromise<T> | ArrayOrIterator<T>;
  *
  * @class Observable<T>
  */
-export class Observable<T> implements CoreOperators<T>  {
+export class Observable<T> {
```

Comment:
>  I don't see a real need for them 

: In general I have similar feeling since I didn't had lot of cases of use interface to observable instead of observable itself but might be some usecases I couldn't think of. cc @blesh for opinions.